css/selectors/media/sound-state.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<title>Sound State: the :muted and :volume-locked pseudo-classes</title>
+<link
+  rel="help"
+  href="https://w3c.github.io/csswg-drafts/selectors/#sound-state"
+/>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<body>
+  <video width="300" height="300" muted loop></video>
+  <script type="module">
+    // Unfortunately, we can't test the volume-locked state because it's not
+    // possible to lock the volume of a video element with JS.
+    
+    test((t) => {
+      for (const pseudo of [":muted", ":volume-locked"]) {
+        try {
+          document.querySelector(`.not-a-thing${pseudo}`);
+        } catch (e) {
+          assert_unreached(`${pseudo} is not supported`);
+        }
+      }
+    }, "Test :pseudo-class syntax is supported without throwing a SyntaxError");
+
+    promise_test(async (t) => {
+      assert_equals(
+        document.querySelector("video:muted"),
+        null,
+        "must know :muted"
+      );
```

Comment:
This fails, I think because the `<video>` has the `muted` attribute. I think you may want to remove that?